samples/spring-petclinic/README.md

```diff
@@ -0,0 +1,32 @@
+
+## Use CAMP with Spring-petclinic application
+
+We provide a sample docker-based project called `Spring-petclinic` which is a popular spring-based web application https://github.com/spring-projects/spring-petclinic that supports different types of DB (HSQLDB, Postgres, MySQL) and Jetty as web sever.
+
+**NB:** Before generating configurations, please edit the `camp.yml` file by giving the host machine ip (use `ifconfig` and provide the `en0` ip).
+```
+pattern: "IP_ADDRESS"
+replacements: [ "XXX.XXX.XXX.XXX" ] #provide your host machine IP
+```
+Generate configurations to support different DB (PetClinic app + MySQL and PetClinic app + Postgres).
+```
+docker run -t -v $PWD:/campworkingdir -v /var/run/docker.sock:/var/run/docker.sock -t fchauvel/camp:latest camp generate -d /campworkingdir
+docker run -t -v $PWD:/campworkingdir -v /var/run/docker.sock:/var/run/docker.sock -t fchauvel/camp:latest camp realize -d /campworkingdir
```

Comment:
I think it would be good to use another camp version than latest. Could you tag the current version and use it. It would avoid issues if we break and change the current version.